lxc/image.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/gosexy/gettext"
+	"github.com/lxc/lxd"
+)
+
+type imageCmd struct{}
+
+func (c *imageCmd) showByDefault() bool {
+	return true
+}
+
+func (c *imageCmd) usage() string {
+	return gettext.Gettext(
+		"lxc image import <filename> [options] [resource:]\n" +
```

Comment:
[resource:] and [options] should be flipped. The spec defines the command as:

    image import <tarball> [target] [--created-at=ISO-8601] [--expires-at=ISO-8601] [--fingerprint=HASH] [prop=value]